.github/workflows/dockerhub-push.yml

```diff
@@ -0,0 +1,38 @@
+name: HMDA Docker Hub Image Push
```

Comment:
General question: what fills in all the template values?

Reply:
Can you clarify this question

Reply:
Where do all the values in curly brackets come from?